src/extension.ts

```diff
@@ -1737,6 +1738,27 @@ const debugAttachKubernetes = async (explorerNode: ClusterExplorerResourceNode)
     }
 };
 
+const selectPodKubernetes = async () => {
+    const resource =  await kubectlUtils.getResourceAsJson<KubernetesCollection<Pod>>(kubectl, "pods");
+    if (!resource) {
+        return;
+    }
+
+    const podPickItems = resource.items.map((pod) => {
+        return {
+            label: `${pod.metadata.name} (${pod.spec.nodeName})`,
```

Comment:
Suggest some text to explain the brackets e.g. 
```suggestion
            label: `${pod.metadata.name} (on ${pod.spec.nodeName})`,
```